system-parachains/collectives/collectives-polkadot/src/ambassador/migration.rs

```diff
@@ -0,0 +1,389 @@
+			(2, "13uuCeyenBHFGU3VNemLEY97WZVSXw9KXv8AJG6VEx32MBfg"),
+			(2, "13C9eaMNJPpEqEKRsEJ9P4DKxkb7XWQzeugEfdJkGdVFQPYh"),
+			(2, "14QLoJzkneWvzWcLKXnnr32h9q66Cp8KyWSQvwhfxqpbNRh5"),
+			(2, "E5kQJ88kBAwzk5ZqG8FbE5dJJg41dRpeg5BTrcskFNagBx2"),
+			(2, "14QRqymDrSbmxcBQnoPeWTfrwcoCkKCFYQfFZfyqNLmsfS4G"),
+			(2, "16ksAUbRPzzAJJmZBR1nYwxsNVsUrkaFPvDS4htLtv9fiU4a"),
+			(2, "12zTcLRLU9LjFSCP5WKqAQqgKCUUrFGd5EVBJ8Br8aoryjDW"),
+			(3, "1hFmn2CuqXqxHgKDqqs2xRBpsPkiRXzJfcLbfDgsW7qgmpA"),
+			(3, "5CXb4GNCFZU5WKxcduG6rLDoyQXX6EFUzEvYYbxvUHCVKz5S"),
+			(3, "155UDCSFsenWzWYDroTcZ28ApxrX8w9qAzE8XLqhsYx8oHmU"),
+			(3, "143faiSeYd3agjdp5PBbg4iuFioeRJyC12ZuGsXJRGetaBNU"),
+			(3, "5F1AJAW5JaZ3DdyPu1Wntn1qiNmBEGGMdGbGpybfbcwigmyC"),
+			(3, "12wnTQnY3qUTCofi792fTdf9HpQQ6jmyxrVdwyT6X2Jed4Yg"),
+			(3, "1682no4EwAfKspFBS5H1Yp3iPsLZTvrPBzR94xuWkyFJ2N3i"),
+			(3, "12WQMLv8itdKJ83R4y3qNGYCRrsU6bFjyd4ZvcE4tmJoSiPv"),
+			(3, "14QVZVSc1T7Gi55EL25LJfN5xDuCkdRNwPda4ZUvwMvJBC2g"),
+			(3, "14QhzKsxmrECK8EDpLzzep8tWcYMUPYfXGfmgqM8cVtYRJjJ"),
+			(3, "5CRxWu1z86Z9o9LGjv2QNXkxKQzRe4e29YYzdnnsQA3qesZb"),
+			(3, "1vYC9dziqQmAqoeduwbkaKMXx2FiwhpDhj7gDAPqM3t1Xyt"),
+			(3, "14rGj5dq7eiGJoDQsBHQhQKVgW71X4WLjV3TmBsJLHBxNw5x"),
+			(3, "1LboBQLsa1iTpGzZvXcSd5VF7jfUYf6MzPNoRy2HT9D9FNk"),
+			(3, "16FjvFZfimuixGF3zuqhmxq8XefzFGh6a2hoBJHpCTW4hJFK"),
+			(3, "13dLY5aynbYDCy1DxQYQGjd4yy5uSeVtA4pLdR7SrcxZyvT1"),
+			(3, "12itHPhSz9YKzydpCBPSxewfAJYZuReRPTP5qSuD5LPSeA41"),
+			(3, "12uMmwxjpbntEkvfHDgb9Kdpeo98LNsj7KRLKMMqq4BdG5Qz"),
+			(3, "14dwjTET5jJyfBELo1t37gr2uR7LGESPpUhSfdWyH9SL9KyF"),
+			(3, "16dUmCQM28nzqAAG96rtLUbL9ni5LyT1wRx4SZoS11qnG94f"),
+			(3, "14wyzJLVcCvjXcBnB4DTYvtbobmri5fjAQQKwaRA3JeCxLos"),
+			(3, "14mH2752oH6phG36st3Zc62vFJD1X2JSdhMoNBoxRPnRAHNJ"),
+			(3, "1eh8ed7Mv4Bv22xTKpfYKJdLCHr5oZwPH24eZmBB8Pzw9AB"),
+			(3, "15MNuKZdQ65waaiRSmdzv2URNiHwprq8WTvYqbZDfpm8W4x6"),
+			(3, "1HomBzZwkpQjhm3e9hQJ2AVFPmV4Ppon9ycCo91bh88xmGn"),
+			(3, "5CLyeaugtNX8E24cFMRJZanhV1EZfTHwkt2MgFmcez9iuEe4"),
+			(3, "123jNGxHk9ZV7oVVhFWFtMghNpmnmmTWxSpNxf8TTKzmCSQ2"),
+			(3, "131cQySF87ceHM9F7JZb6rRfaGzGvX42oJKty6deyCKc5TRh"),
+			(3, "143hSsxzuTfEsmbRBgdP1uc78nkgFN8z77ikT598bGs5ueMA"),
+			(3, "1jPw3Qo72Ahn7Ynfg8kmYNLEPvHWHhPfPNgpJfp5bkLZdrF"),
+			(3, "15wb37SKZ49jaTTtgz8312jKcWb1EwRTX3sDkJ1cs4RB97Rh"),
+			(3, "13RBN6UF43sxkxUrd2H4QSJccvLNGr6HY4v3mN2WtW59WaNk"),
+			(4, "135p4GMyXz4gGT4969k9kP1wR4KfURZrVoFZt2CDCWxxxpCm"),
+			(4, "15rYBV5YwGmhzee5PWqrnQtb2HhwWP2rK2f4cLMhFfcNdPZL"),
+			(4, "11Asf5YWRX1HGtTh1GcbtuaeYgbAJUWtSHCHaY23zgEeiit"),
+			(4, "13QdCrHmSk2AnNgaYLAK8Mq9fdyvrouckBXTw5ccfJK8kkCS"),
+		];
+
+		for (index, val) in ambassador_ss58.iter().enumerate() {
+			let account: AccountId32 = <AccountId as Ss58Codec>::from_string(val.1).unwrap();
+			let account32: [u8; 32] = account.clone().into();
+			assert_eq!(addresses[index].0, ambassador_ss58[index].0, "ranks must be equal.");
+			assert_eq!(addresses[index].1, account32, "accounts must be equal.");
```

Comment:
You could also directly call the `on_runtime_upgrade` once and then assert the post-ranks.